src/templates/normal-catalog.tsx

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+import { Card, Divider, Flex } from "antd";
+import { graphql } from "gatsby";
+import Layout from "../components/Layout";
+import Content, { HTMLContent } from "../components/shared/Content";
+import { GatsbyImage, getImage } from "gatsby-plugin-image";
+import { FileNode } from "gatsby-plugin-image/dist/src/components/hooks";
+import NormalCellLines from "../component-queries/NormalCellLines";
+
+const {
+    coriellCard,
+    banner,
+    bannerContent,
+    header,
+    mainHeading,
+    coriellWrapper,
+} = require("../style/catalog.module.css");
+interface NormalCatalogTemplateProps {
+    title: string;
+    content: string;
+    contentComponent?: JSX.ElementType;
+    footerText: string;
+    main: {
+        heading: string;
+        description: string;
+        subheading: string;
+    };
+    coriellImage: FileNode;
+    coriellLink: string;
+}
+// eslint-disable-next-line
+export const NormalCatalogTemplate = ({
+    title,
+    content,
+    contentComponent,
+    footerText,
+    main,
+    coriellImage,
+    coriellLink,
+}: NormalCatalogTemplateProps) => {
+    const image = getImage(coriellImage);
+    const PageContent = contentComponent || Content;
+    return (
+        <section>
+            <h1 className="">{title}</h1>
+            <Flex className={header}>
+                <PageContent className="content" content={content} />
+                <Divider
+                    type="vertical"
+                    style={{ height: "initial", marginInline: "20px" }}
+                />
+                <div className={coriellWrapper}>
+                    {image && (
+                        <a href={coriellLink} target="_blank" rel="noreferrer">
+                            <Card
+                                bordered={true}
+                                className={coriellCard}
+                                title="View Allen Cell Collection on"
+                                cover={
+                                    <GatsbyImage image={image} alt="Coriell" />
+                                }
+                            ></Card>
+                        </a>
+                    )}
+                </div>
+            </Flex>
+            <h2 className={mainHeading}>{main.heading}</h2>
+            <Card className={banner} bordered={true}>
+                <h4>{main.subheading}</h4>
```

Comment:
Nit: For accessibility, heading elements should follow chronological hierarchy. Consider making this an `h3` element and overriding the styling!

Reply:
i was not aware of this (same with the table header) and the headers are used for consistent typography across the app. I would prefer to address this all at once than have some places the styling is overwritten.